Makefile

```diff
@@ -92,11 +92,11 @@ LDFLAGS := -fp hard -nodefaults
 ifeq ($(GENERATE_MAP),1)
 	LDFLAGS += -map $(MAP)
 endif
-
 CFLAGS  = -msgstyle gcc \
 		-cwd source \
 		-Cpp_exceptions off \
 		-DGEKKO \
+		-DMWERKS_GEKKO \
```

Comment:
Just a shorthand for `__MWERKS__ && GEKKO`, regardless of whether the target is meant to match.

Reply:
This can be done with
```c
#if defined(__MWERKS__) && defined(GEKKO)
#define MWERKS_GEKKO
#endif
```
in a header somewhere

Reply:
I did that at first but `clang` (and I imagine other compilers) complain that "expanding `defined() in a preprocessor directive is undefined behavior."

Reply:
Oh wait, nevermind. That would work. Is it better?

Reply:
Yeah I think in general it's a good idea to push this kind of logic out of the makefile and into the sources